variables.tf

```diff
@@ -337,3 +337,9 @@ variable "node_resource_group" {
   type        = string
   default     = null
 }
+
+variable "only_critical_addons_enabled" {
+  description = "(Optional) Enabling this option will taint default node pool with CriticalAddonsOnly=true:NoSchedule taint. Changing this forces a new resource to be created."
```

Comment:
Would the following code be better to match the document in the provider?

```hcl
description = "(Optional) Enabling this option will taint default node pool with `CriticalAddonsOnly=true:NoSchedule` taint. Changing this forces a new resource to be created."
```

Reply:
@lonegunmanb updated with markdown formatting